### PR TITLE
Update CI/CD documentation

### PR DESCRIPTION
**Pull request by juliamrch**
## Describe your PR

_Summarize your changes here : explain what, how, and why. Be as explicict as you can on why your changes are needed and avoid implicit reasoning._

The CI/CD page have been converted into a section and the documentation have been updated.

This PR creates a sub level for the CI/CD section, with the following changes in the CI/CD section:

- [A dedicated page for GitHub](http://doc-review-pr-233.cleverapps.io/doc/ci-cd/github/)
- [A dedicated page for GitLab with updated info for Heptapod](http://doc-review-pr-233.cleverapps.io/doc/ci-cd/gitlab/)
- [A dedicated page for writing a custom CI/CD with the CLI with a link to predefined variables](http://doc-review-pr-233.cleverapps.io/doc/ci-cd/custom-scripts/)
- A new partial for mandatory CI/CD configuration 

Additionally, [the CI/CD components catalog have been added to Heptapod features](http://doc-review-pr-233.cleverapps.io/doc/addons/heptapod/#integrated-cicd).

## Checklist

- [ ] My PR is related to an opened issue : #
- [x] I've read the [contributing guidelines](../CONTRIBUTING.md)

## Reviewes
_Who should review these changes?_ @

### CODE REVIEW

File: content/doc/ci-cd/gitlab.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+For example, to deploy any commit on your production app, use:
+
+```yaml
+include:
+  - component: $CI_SERVER_HOST/$CI_PROJECT_PATH/deploy-to-prod@~latest
+```
+
+This assumes you have a running app on Clever Cloud. Any commit on your default branch (`main`, `master` or other name) triggers a deployment for this app. [Inject the following variables in your GitLab repository settings](https://docs.gitlab.com/ee/ci/variables/index.html#for-a-project):
+
+- `APP_ID`: you app_id on Clever Cloud, find it at the top right in Clever Cloud Console, in your application tab.
```
Comment (davlgd) [inline content/doc/ci-cd/gitlab.md:37]:
```suggestion
- `APP_ID`: Find it at the top right in [Clever Cloud Console](https://console.clevercloud.com), in your application tab.
```

File: content/doc/ci-cd/gitlab.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- `CLEVER_TOKEN` and `CLEVER_SECRET`
+
+### Deploy from a self-hosted GitLab instance
+
+You can use pipeline components to deploy from a self-hosted GitLab instance, by including it as a project and mirroring the Pipeline repository. Follow [GitLab documentation](https://docs.gitlab.com/ee/ci/components/#use-a-gitlabcom-component-in-a-self-managed-instance) to get full instructions.
```
Comment (davlgd) [inline content/doc/ci-cd/gitlab.md:42]:
```suggestion
You can use pipeline components to deploy from a self-hosted GitLab instance, by including it as a project and mirroring the pipeline repository. Follow [GitLab documentation](https://docs.gitlab.com/ee/ci/components/#use-a-gitlabcom-component-in-a-self-managed-instance) to get full instructions.
```

File: content/doc/ci-cd/gitlab.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Deploy directly from Heptapod
+
+The self-hosted GitLab service on Clever Cloud with [Heptapod](/doc/addons/heptapod) already hosts the components, you don't need to do any set up like you would in a self-hosted instance.
```
Comment (davlgd) [inline content/doc/ci-cd/gitlab.md:46]:
```suggestion
The self-hosted GitLab service on Clever Cloud with [Heptapod](/doc/addons/heptapod) already hosts the components, you don't need to do any set up like you would in a self-hosted instance:
```

File: layouts/shortcodes/content/ci-cd-configuration.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+
+### Mandatory configuration
+
+Your repository must contain at least the following variables to use the CLI:
+
+- `CLEVER_TOKEN`
+- `CLEVER_SECRET`
+
+Find it in your machine, usually in `~/.config/clever-cloud/clever-tools.json`, after [installing the CLI](/doc/CLI).
```
Comment (davlgd) [inline layouts/shortcodes/content/ci-cd-configuration.md:9]:
```suggestion
Find it in your machine, usually in `~/.config/clever-cloud/clever-tools.json`, after installing [Clever Tools](https://github.com/CleverCloud/clever-tools/).
```

**Review by davlgd — CHANGES_REQUESTED** [review]
LGTM after some changes, great new pages, congrats 👏

**Review by cnivolle — APPROVED** [review]

**Pull request merged**